.github/workflows/automated-tests.yml

```diff
@@ -16,20 +16,124 @@ on:
 permissions:
   contents: read
 jobs:
-  build-install-and-test:
+  build-bbb-apps-akka:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
       - run: ./build/get_external_dependencies.sh
       - run: ./build/setup.sh bbb-apps-akka
+      - run: tar cvf artifacts.tar artifacts/
+      - name: Archive packages
+        uses: actions/upload-artifact@v3
+        with:
+          name: artifacts_bbb-apps-akka.tar
+          path: |
+            artifacts.tar
+  build-bbb-config:
+    runs-on: ubuntu-20.04
```

Comment:
note to whoever is merging into `develop` to switch to ubuntu 22.04

Reply:
It's targeting 2.7.
Once it's merged. We merge 2.7 into Develop and tweak Ubuntu version.
Thanks for reminding me.